builder/processor-tools/src/main/java/io/helidon/builder/processor/tools/BeanUtils.java

```diff
@@ -110,10 +123,28 @@ public static boolean validateAndParseMethodName(String methodName,
      * @return true if it appears to be a reserved word
      */
     public static boolean isReservedWord(String word) {
-        word = word.toLowerCase();
-        return word.equals("class") || word.equals("interface") || word.equals("package") || word.equals("static")
-                || word.equals("final") || word.equals("public") || word.equals("protected") || word.equals("private")
-                || word.equals("abstract");
+        word = word.toUpperCase();
```

Comment:
Use switch, or Set.contains (prepare a constant set with reserved words).
Also the `equalsIgnoreCase` is useless there (probably an omission when refactoring)

Reply:
pushed